featured-single-post.php

```diff
@@ -1,45 +1,72 @@
 <?php
 /**
  * Template Name: Featured Post
- * Template Post Type:
+ * Template Post Type: Post
  */
 disallow_direct_load( 'featured-single-post.php' );
 get_header( 'featured' ); the_post();
 
+$title = $post->post_title;
+$cats = wp_get_post_categories($post->ID);
+$category_title = get_cat_name($cats[0]);
 $subtitle = get_post_meta( $post->ID, 'subtitle', TRUE );
+$deck = get_post_meta( $post->ID, 'deck', TRUE );
+$source = get_post_meta( $post->ID, 'source', TRUE);
+$img_attach = get_img_html($post->ID, 'story_feature', Array('return_id' => True));
+
+if($img_attach['attachment_id'] != '') {
+	$attachment = get_post($img_attach['attachment_id']);
+}
+
+$video_url = get_video_url($post->ID);
 
 ?>
-<div class="featured-story-header">
-	<div class="row-fluid">
-		<div class="span5">
-			<?php the_post_thumbnail( null, array( 'class' => 'img-responsive' ) ); ?>
-		</div>
-		<div class="span7">
-			<div id="page-title" class="featured-title">
-				<h1><?php echo the_title(); ?></h1>
-				<?php if ( $subtitle ) : ?>
-				<p id="subtitle"><em><?php echo $subtitle; ?></em></p>
-				<?php endif; ?>
-				<div id="deck" class="featured-deck">
-					<p class="lead"><?php the_excerpt(); ?></p>
+
+<div id="feature-story">
+	<div class="container">
+		<div class="row">
+			<div class="span12">
+				<div class="feature-headlines">
+					<h2 class="category-title"><?php echo $category_title; ?></h2>
+					<h1 class="story-title"><?php echo the_title(); ?></h1>
+					<?php if ( $subtitle ) : ?>
+					<h3 class="subtitle"><?php echo $subtitle; ?></h3>
+					<?php endif; ?>
+					<?php echo do_shortcode( '[feature_post_meta css="clearfix"]' ); ?>
 				</div>
-			</div>
-			<hr>
-			<div id="by-line" class="featured-byline">
-				<?php echo do_shortcode( '[single_post_meta css="clearfix"]' ); ?>
+				<?php if($video_url != '') : ?>
+					<?=$wp_embed->run_shortcode( '[embed width="550" height="500"]'.$video_url.'[/embed]' )?>
```

Comment:
Avoid using shorthand php tags (`<?=`) in new code; `<?php echo` would be preferable here instead.